docs/jwst/rscd/reference_files.rst

```diff
@@ -1,92 +1,108 @@
 
    corrected data = input data data + dn_accumulated * scale * exp(-T / tau)  (Equation 1)
 
-where T is the time since the last group in the previous integration, tau is the exponential time constant found in the RSCD table
-and  dn_accumulated is the DN level that was accumulated for the pixel from the previous integration.
-In case where the last integration  does not saturate the :math:`scale` term in equation 1 is determined according to the equation:
+where:
+
+ - T is the time since the last group in the previous integration
+ - tau is the exponential time constant found in the RSCD table
+ - dn_accumulated is the DN level that was accumulated for the pixel from the previous integration.
+
+In cases where the last integration does not saturate, the :math:`scale` term in equation 1
+is determined according to:
 
   :math:`scale = b{1}* [Counts{2}^{b{2}} * [1/exp(Counts{2}/b{3}) -1 ]\; \; (Equation \; 2)`
 
 The following two additional equations are used in Equation 2:
 
   :math:`b{1} = ascale * (illum_{zpt} + illum_{slope}*N + illum2* N^2) \; \; (Equation \; 2.1)`
-  :math:`Counts{2} = Final \, DN \, in \, the \,  last \, group \, in \; the \, last \, integration 
-      \, - Crossover \, Point \; \; (Equation \; 2.2)`
+
+  :math:`Counts{2} = Final \, DN \, in \, the \,  last \, group \, in \; the \, previous \, integration \, - Crossover \, Point \; \; (Equation \; 2.2)`
       
-The parameters for equations 2, 2.1, and 2,2  are:
-  - :math:`b{2}` in equation 2 is table column POW from RSCD table
-  - :math:`b{3}` in equation 2 is table column  PARAM3 from the RSCD table
-  - ascale  in equation 2.1 is in the RSCD table 
-  - :math:`illum_{zpt}`  in equation 2.1 is in the RSCD table 
-  - :math:`illum_{slope}`  in equation 2.1 is in the RSCD table
-  - :math:`illum2`  in equation 2.1 is in the RSCD table
-  - N  in equation 2.1 is the number of groups per integration
-  - Crossover Point in equation 2.2 is CROSSOPT in the RSCD table
-
-If the previous integration saturates, :math:`scale` is no longer calculated using equation 2 - 2.2, instead it is calculated 
-using equations 3 and 3.1.
+where:
+
+ - N in equation 2.1 is the number of groups per integration
+ - Crossover Point in equation 2.2 is column CROSSOPT in the RSCD table.
+
+If the previous integration saturates, :math:`scale` is no longer calculated using equations
+2 - 2.2. Instead it is calculated using equations 3 and 3.1:
 
    :math:`scale_\text{sat} = slope * Counts{3} + sat_\text{mzp} \; \; (Equation \; 3)`
  
-   :math:`slope = sat_{zp} + sat_{slope} * N + sat_2*N^2 + evenrow_{corrections} \; \; (Equation \; 3.1)`.
+   :math:`slope = sat_{zp} + sat_{slope} * N + sat_2*N^2 + evenrow_{corrections} \; \; (Equation \; 3.1)`
 
-The parameters in equation 3 and 3.1 are:
+where:
 
-    - :math:`Counts{3}` in equation 3 is an estimate of the what the last group in the
+    - :math:`Counts{3}` is an estimate of the what the last group in the
```

Comment:
Since the RSCD table entries have changed to lower case we should change the reference of RSCD table values to lower case so in 
:math: `scale_\text{sat}` is SAT_SCALE in RSCD table - should not SAT_SCALE be sat_scale. Likewise for the other cases

Reply:
The above comment is meant for the code below